ballerina/tests/listener_test.bal

```diff
@@ -108,7 +110,7 @@ function testCreateRecord() {
 }
 
 @test:Config {
-    enable: false,
+    enable: true,
```

Comment:
No need to keep the `enable` property if the test is enabled. Apply where necessary
```suggestion
```